packages/platform-web/src/node/features/html-render/lib/getPageMiddleware.ts

```diff
@@ -10,21 +10,32 @@ import { IHandlePageRequest, RequestContext, ISendHtml } from '../serverHooks';
 import { renderToHTML } from './renderToHTML';
 
 function createPageHandler(serverPluginContext: IServerPluginContext) {
+  const {
+    traces: { serverRequestTrace }
+  } = serverPluginContext;
   const wrappedSendHtml = async (
     html: string,
     { req, res }: RequestContext
   ) => {
+    const sendHtmlOriginalTrace = serverRequestTrace.traceChild(
+      'SHUVI_SERVER_SEND_HTML_ORIGINAL'
```

Comment:
We should define all these `SHUVI_SERVER_SEND_HTML_ORIGINAL` as constants in a place, So we can get a clear picture of all traces we have.

Reply:
Done.
In fact, it was placed in constants at the first version, and then there were some field adjustments later, and I found that next.js did not store them in constants, so I deleted it 😂